imports/ui/QuestionEditItem.jsx

```diff
@@ -542,12 +542,12 @@ export class QuestionEditItem extends Component {
                 </div>
               </div>
               <div className='col-md-6'>
-                <select value = {this.state.courseId ? this.state.courseId : -1} className='ql-header-button question-type form-control pull-right' onChange={this.setCourse}>
+                <select value={this.state.courseId ? this.state.courseId : -1} className='ql-header-button question-type form-control pull-right' onChange={this.setCourse}>
```

Comment:
don't we want spaces around operators?

Reply:
This is another Standard.js rule. Considering that you usually don't include spaces with HTML attribute-value pairs, I think it make sense.